docs/extensibility/50-list-and-details-widgets.md

```diff
@@ -578,6 +581,64 @@ See the following example:
 
+### StatisticalCard
+
+StatisticalCard widgets render a card component with a several numerical information elements.
+This widget is primarily designed to be used in Monitoring and Health section **data.details.health** or via [injections](#widget-injections-overview) (**destination: ClusterOverview, slot: health**), allowing the card to be rendered within the dense grid layout of the ClusterOverview's Monitoring and Health section.
+
+These are the available `StatisticalCard` widget parameters:
+
+| Parameter            | Required | Type                                       | Description                                                                                                                                                                                                          |
+| -------------------- | -------- | ------------------------------------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **mainValue**        | **Yes**  | object                                     | Main value displayed with bigger font                                                                                                                                                                                |
+| **mainValue.source** | **Yes**  | string or [JSONata](jsonata.md) expression | It is used to fetch data for the column. In its simplest form, it's the path to the value.                                                                                                                           |
+| **mainValue.name**   | **Yes**  | string                                     | Name for the primary label of this field. Required for most widgets (except for some rare cases that don't display a label). This can be a key to use from the [**translation** section](./translations-section.md). |
+| **children**         | No       | array of objects                           | Array of additional values, listed next to the main one                                                                                                                                                              |
+| **children.source**  | **Yes**  | string or [JSONata](jsonata.md) expression | It is used to fetch data for the column. In its simplest form, it's the path to the value.                                                                                                                           |
+| **children.name**    | **Yes**  | string                                     | Name for the primary label of this field. Required for most widgets (except for some rare cases that don't display a label). This can be a key to use from the [**translation** section](./translations-section.md). |
```

Comment:
```suggestion
| **children.name**    | **Yes**  | string                                     | The name for the primary label of this field. Required for most widgets (except for some rare cases that don't display a label). This can be a key to use from the [**translation** section](./translations-section.md). |
```